plugins/woocommerce/includes/class-wc-session-handler.php

```diff
@@ -210,14 +330,14 @@ public function maybe_set_customer_session_cookie() {
 	 */
 	public function set_customer_session_cookie( $set ) {
 		if ( $set ) {
-			$to_hash           = $this->_customer_id . '|' . $this->_session_expiration;
-			$cookie_hash       = hash_hmac( 'md5', $to_hash, wp_hash( $to_hash ) );
-			$cookie_value      = $this->_customer_id . '||' . $this->_session_expiration . '||' . $this->_session_expiring . '||' . $cookie_hash;
-			$this->_has_cookie = true;
+			$cookie_hash  = $this->hash( $this->_customer_id . '|' . $this->_session_expiration );
+			$cookie_value = $this->_customer_id . '|' . $this->_session_expiration . '|' . $this->_session_expiring . '|' . $cookie_hash;
```

Comment:
Any idea why `||` was used? Would it matter that we invalidate existing cookies? I think that's fine, they expire in 2 days anyway.

Reply:
No valid reason. There is a separate issue for this, I just applied the fix here while I was touching the file. Can be addressed separately and rebased later.

Reply:
Fine to leave it